scanomatic/qc/phenotype_results.py

```diff
@@ -1207,26 +1212,26 @@ def _animation():
         index = 0
         while index < plate.shape[-1]:
             im.set_data(plate[..., index])
-            ax.set_title("Time {0}".format(index))
+            _ax.set_title("Time {0}".format(index))
             index += 1
 
             yield
 
-    return _animation()
+    return _animation(ax)
```

Comment:
have you tested that this works? I Thought I attempted it but there was some issue

Reply:
I think theres a readme or similar how to apply it to any function that plots and updates a figure

Reply:
I changed this because VSC was unhappy with using ax the way it was used. To me it doesn't look suspicious (but on the other hand it didn't really look suspicious before my change either), but I will try it out.

Reply:
Reverterade delvis. Se koden.